File: apiEZPZ/src/main/java/com/example/apiezpz/prohibited/domain/Detection.java
package com.example.apiezpz.prohibited.domain;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
@Table(name = "detection")
public class Detection {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String airportName; // 공항명 (청주공항 등)

    private String category; // 적발된 물품 유형 (날카로운 물체, 공구류 등)

    private int detectionCount; // 적발 건수
}

File: apiEZPZ/src/main/kotlin/com/example/apiezpz/prohibited/domain/Detection.kt
package com.example.apiezpz.prohibited.domain

import jakarta.persistence.*

@Entity
@Table(name = "detection")
data class Detection(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,

    var airportName: String? = null, // 공항명 (청주공항 등)

    var category: String? = null, // 적발된 물품 유형 (날카로운 물체, 공구류 등)

    var detectionCount: Int = 0 // 적발 건수
) 